data/hai/hai reveal 1 intro.txt

```diff
@@ -58,11 +58,49 @@ mission "Event: Pug Media Coverage"
 		event "hai secret leaks" 10 15
 		log "Minor People" "Maduena Kabiru" `A senator from Bounty in the Delta Velorum system. He is somewhat paranoid, and insists upon the Navy taking greater steps to better explore the galaxy beyond human space to search for possible threats.`
 		conversation
-			`As you walk through the spaceport, you can't help but notice that coverage on the events of the Pug invasion still continues to dominate headlines, even though it's been weeks now.`
-			`	Looking around, you see that experts and officials have given way to talking heads and theorists. Some of them are saying what you also suspect: that the Pug invasion was a ploy to cause humanity to band together, learn to resolve differences without conflict, and ensure the sorts of extremism that led to the events precipitating the war would not be tolerated.`
-			`	But others...`
-			`	One particular paranoid fringe-world politician dominates the air on the channel you're watching here. Councilor Maduena Kabiru from Bounty demands that the edges of human space be better understood, that threats and even potential threats must clearly exist beyond what is already known, and that it is the responsibility of the government to find out and inform the people. While one could definitely argue in favor of his point, it doesn't carry much weight coming from a councilor whose system is a mere two jumps from a brand new Navy base.`
-			`	In this very public space you spot a couple of people who seem to hesitantly recognize you, presumably from other coverage, and so you keep your thoughts to yourself.`
+			`As you leave your ship, you see two captains ranting about a video playing on a datapad. The walkway forces you to pass close to them.`
+			choice
+				`	"Hey! What's this video about?"`
+				`	(Leave them be.)`
+					decline
+			`	You're startled when the captains turn to you, both with preposterous sideburns and rat-like mustaches. It's impossible to tell if they're smiling or frowning, as the pair live squarely in the middle of the uncanny valley. They appear to be twins, as are their well-armed Headhunters.`
+			`	The left twin asks, "Can you believe this? It's been weeks and everyone's still obsessing over the Pug invasion. The news was all experts and officials at first, but now we just see talking heads and conspiracy theorists."`
+			`	In disbelief, the right twin responds, "It's just a propaganda exercise to boost Syndicate sales. Even if these elusive 'Pug' existed, we won't hear the end of it as long as the story boosts public opinion of the Syndicate."`
+			choice
+				`	"The past few months have brought some of the biggest changes in recent history. Of course people are going to keep talking about it."`
+				`	"Don't you see this is what we need? The Pug invasion was a ploy to cause humanity to band together and learn to resolve differences without conflict."`
+				`	"Oh, the Pug were real alright. I met some and fought them."`
```

Comment:
You do in both branches of FW. The Pug notice your ship and flag you down to do some sort of oath to them because they noticed you're from the outside or something. 

I'm wondering how many of the developers have actually played the campaign, which could explain a couple of things...